File: SDKs/orthanc-server.com/1.9.6/typescript/src/sdk/models/operations/getinstancesidmatlab.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";


export class GetInstancesIdMatlabPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: string;
}


export class GetInstancesIdMatlabRequest extends SpeakeasyBase {
  @Metadata()
  pathParams: GetInstancesIdMatlabPathParams;
}


export class GetInstancesIdMatlabResponse extends SpeakeasyBase {
  @Metadata()
  body?: Uint8Array;

  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;
}

File: SDKs/orthanc-server.com/1.9.6/python/src/sdk/models/operations/get_instances_id_matlab.py
from dataclasses import dataclass, field
from typing import Optional


@dataclass
class GetInstancesIDMatlabPathParams:
    id: str = field(metadata={'path_param': { 'field_name': 'id', 'style': 'simple', 'explode': False }})
    

@dataclass
class GetInstancesIDMatlabRequest:
    path_params: GetInstancesIDMatlabPathParams = field()
    

@dataclass
class GetInstancesIDMatlabResponse:
    content_type: str = field()
    status_code: int = field()
    body: Optional[bytes] = field(default=None)
    
